.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
```

Comment:
You can click delete file from the Files changed section.
![image](https://user-images.githubusercontent.com/53231968/96677007-44363e00-13a1-11eb-8a59-27fde1530198.png)